### DISCUSSION TITLE
Custom workouts not working on Chrome (MacOs)

### DISCUSSION THREAD

**Discussion by Jelly-Legs** [dvmarinoff/Auuki#154] [Q&A]
I tried uploading .gpx and .fit files but nothing happened after the file had been selected. Am I missing something or this is a bug??

![image](https://github.com/dvmarinoff/Flux/assets/146718928/7ab3f34e-a876-4d4c-bfe3-cfde7230a023)

**Comment by dvmarinoff** [comment]
Hi and welcome to the project!

You are not missing anything, it's just that gpx files and .fit workouts are not supported. The project supports only .zwo (Zwift format) workouts and .fit Course files. Thanks to your feedback I am seeing now that I haven't made this clear in the app itself. So I just added a description under the load button to note that point. Here is how things look now, with 3 example uploads:
- .fit activity file,
- .zwo file creted with zwofactory.com and,
- .zwo file created with the Flux build-in zwo editor

<img width="732" alt="Screenshot 2023-10-02 at 15 21 23" src="https://github.com/dvmarinoff/Flux/assets/7523832/80b963a6-858e-43fd-8f9f-789231049d26">


Thanks for the feedback!
